projects/angular-calendar/schematics/ng-add/index.ts

```diff
@@ -113,7 +115,7 @@ function addModuleToImports(options: Schema): Rule {
     const moduleName = `CalendarModule.forRoot({ provide: DateAdapter, useFactory: ${
       options.dateAdapter === 'moment'
         ? 'momentAdapterFactory'
-        : 'adapterFactory'
+        : (options.dateAdapter === 'luxon' ? 'luxonAdapterFactory' : 'adapterFactory')
```

Comment:
Let's bake this into the `dateAdapters` object and avoid the branching logic e.g.

```
const dateAdapters: Record<Schema['dateAdapter'], {version: string, adapterName: string}> = {
      moment: {
        version: momentVersion,
        adapterName: 'momentAdapterFactory'
      },
      'date-fns': {
        version: dateFnsVersion,
        adapterName: 'adapterFactory'
      },
      luxon: {
				version: luxonVersion,
				adapterName: 'luxonAdapterFactory'
			},
    };
```

Then here we can just pass dateAdapters[options.dateAdapter].adapterName;